recipes/dr-disco/meta.yaml

```diff
@@ -13,7 +13,7 @@ source:
 
 build:
   preserve_egg_dir: true
-  number: 0
+  number: 1
```

Comment:
`htslib` pinned to fixed version

Reply:
we should not fix those in a bulk migration. This should be fixed separately or with the next update. The maintainer probably has some reasons for this?

Reply:
Oh, sorry, my comment on this review were not very precise. Not all of them are meant as change request and others only point at reasons why something else should or shouldn't be changed. Sorry for being so imprecise!

So here it's just that `htslib` is not dependent on `CONDA_HTSLIB` and hence the build number shouldn't be changed, i.e., this packages needn't be part of this PR.